[14.0][FIX] cetmix_tower_server: Clean up Python command execution result

dessanhemrayev opened the pull request:
Renamed method _parse_ssh_command_results to  _parse_command_results for using not only for SSH but for any command including Python one.

Task: 3952

dessanhemrayev committed:
[FIX] cetmix_tower_server: Clean up Python command execution result

Renamed method _parse_ssh_command_results to  _parse_command_results for using not only for SSH but for any command including Python one.

Task: 3952

ivs-cetmix commented on cetmix_tower_server/tests/test_command.py:
```diff
@@ -837,3 +844,27 @@ def test_execute_python_code(self):
             "Error in command result must be set to None",
         )
+
+    def test_execute_python_code_with_secret(self):
```
We need to check if secret key is replaced in the command output. I didn't manage to find such a flow in this test.

dessanhemrayev replied:
Thanks, I'll fix it

ivs-cetmix commented on cetmix_tower_server/tests/test_command.py:
```diff
@@ -41,6 +41,19 @@ def setUp(self, *args, **kwargs):
+
+        self.command_create_new_command_2 = self.Command.create(
+            {
+                "name": "Create new command",
```
Command name as well as the name of the variable it's assigned don't look relevant. Which degraded the code readability.
There should be something like "Python command with secret" or similar.

dessanhemrayev replied:
Thanks, I'll fix it

dessanhemrayev committed:
[FIX] cetmix_tower_server: Clean up Python command execution result

Fix _replace_with_spoiler() method to handle key_value with quotes

- Resolved an issue where key_value containing quotes was not correctly processed.
- Modified the method to remove quotes before performing the replacement.

Update tests

GabbasovDinar commented on cetmix_tower_server/models/cx_tower_key.py:
```diff
@@ -450,6 +450,9 @@ def _replace_with_spoiler(self, code, key_values):
         for key_value in key_values:
+            # If key_value includes quotes, remove them for the replacement
+            if key_value.startswith('"') and key_value.endswith('"'):
+                key_value = key_value[1:-1]
```
```suggestion
            key_value = key_value.strip('"')
```

dessanhemrayev committed:
Remove quotes

Co-authored-by: Dinar <git.diga@gmail.com>
Signed-off-by: dessanhemrayev <43703214+dessanhemrayev@users.noreply.github.com>
